README.md

```diff
@@ -1,16 +1,63 @@
 # Exercises
 
-__NOTE:__ DO NOT USE THE BUILT IN METHODS WHEN ASKED TO MAKE THEM! 
+__NOTE:__ DO NOT USE THE BUILT IN METHODS WHEN ASKED TO MAKE THEM!
 
 1. Using Array.prototype add a new method `myForEach` that mimics the behaivor of the built in `forEach`.
 
-2. Use your `myForEach` to add `myMap` to the Array prototype. `myMap` should behave the same as regular `map`. 
+Array.prototype.myForEach = function(callback) {
+for(let i = 0; i < names.length; i++) {
+   callback(this[i]);
+}
+}
+Array(1,2,3)
 
-3. Write a 'myFilter` that behave's the same as `filter`. 
+2. Use your `myForEach` to add `myMap` to the Array prototype. `myMap` should behave the same as regular `map`.
+
+Array.prototype.myMap = function(callback) {
+  let output = []
+for(let i = 0; i < arr.length; i++) {
+    callback(output.push(this[i]));
```

Comment:
ah, okay, I see what went wrong here. You want to call the callback on the element before you push into output. So it'd be `output.push(callback(this[i]))`